docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+ - `longTask` - Command used to provide metrics about [long tasks](/experience/longtasks/).
+ - `elementTiming` - Command used to provide metrics about [element timing](/experience/element-timing/).
+ - `vital-metric` - Command used to provide metrics about [web vitals](/experience/webvitals/).
+
+### Global Variables
+
+The RUM script uses global constants defined in the `constants.js` file. The file contains the following constants:
+
+ - `GLOBAL_KEY` - The global key under which the script is available.
+ - `DEBUG_PARAM_KEY` - When set to `true` turns on the debug level logging. 
+ - `SESSION_COOKIE` - Name of the cookie related to a user session. 
+ - `USER_COOKIE` - Name of the cookie related to the user.
+ - `SESSION_KEEP_ALIVE_MINUTES` - The maximum time for the session to be kept alive.
+ - `PAGELOAD_HARD_TIMEOUT` - Timeout after which the events are no longer considered to be part of the hard page load.
+ - `PAGELOAD_SOFT_TIMEOUT` - Timeout after which the events are no longer considered to be part of the soft page load.
+ - `BOT_USER_AGENT` - Regex used for bots identification.
+
+## Installation
+
+Installation of the RUM script contains the following steps:
+
+### Adding the Loader Script
+
+To add the loader script responsible for loading the main RUM script you need to add the RUM loader snippet generated via the `yarn generate-snippet` command to the header of the page before the closing `</head>` tag:
+
+```
+<script type="text/javascript">
+  ...
+  INSERT SCRIPT SNIPPET HERE
+  ...
+  (window,document,"script","/rum.js","strum");
+</script>
+```
+
+The `/rum.js` should be pointing to the location of the built RUM script and the `strum` is the key under which the script will be available.
+
+### Configuring the RUM Script
+
+To configure the script the `config` command needs to be sent providing the URL of the receiver which accepts the data and the token, which identifies the Sematext Experience application. For example:
+
+```
+strum('config', { token: '8763d12d-1j3t-932v-b498-544290z98k43', receiverUrl: 'https://rum-receiver.sematext.com' });
+```
+
+### Integrating the RUM Script
+
+For some web technologies, especially when it comes to single-page applications additional integration may be needed. Have a look at the [Sematext Experience Integrations](/experience/integrations/) page to learn more.
+
+## Contributing
+
+Pull requests for bug fixes, improvements and new features for the [Browser SDK](https://github.com/sematext/browser-sdk) are more than welcome. You can open a pull request on the [project Github page](https://github.com/sematext/browser-sdk). When opening a new pull request please take the time to briefly describe the changes. Make sure that the newly introduced code passes the `lint` and `flow` checks along with the integration tests. Once the PR is submitted Sematext will review and merge your changes.
+
+## Learn More
+
+Learn more about [Sematext Experience](/experience/). 
```

Comment:
Hope this link works correctly.  You want to point to Experience HP or Experience docs?

Reply:
Changed the link to `https://sematext.com/experience` to bring users to `sematext.com`.